apb.yml

```diff
@@ -43,7 +43,7 @@ plans:
       type: string
       title: URL of the shared service
     bind_parameters:
-    - name: service
+    - name: clientId
       required: True
-      title: The service the client is for
+      title: Service name or Client ID
```

Comment:
I think we need more params than this. We will want to be able to create a public client and a bearer client. So I think we will need to add

```yaml
  - name: clientType
  - required: true
  - title: "the client types are public or bearer" (note we will need to validate it is one of these types in the APB)
  - name: clientId
       required: true 
      title: Service name or Client ID the client is for
```

Reply:
@maleck13 I think we need `clientType` to be dropdown, and not let user to pass it as string

Reply:
If that is possible sounds great

Reply:
I think it is possible, did it yesterday but couldn't test it because of problems with minishift. Will try again today with @witmicko